universal-login-sdk/src/api/sdk.ts

```diff
@@ -209,9 +210,9 @@ class UniversalLoginSDK {
     return this.relayerApi.cancelConnection(authorisationRequest);
   }
 
-  subscribe(eventType: string, filter: any, callback: Function) {
+  subscribe(eventType: WalletEventType, filter: WalletEventFilter, callback: WalletEventCallback) {
     ensure(['KeyAdded', 'KeyRemoved'].includes(eventType), InvalidEvent, eventType);
```

Comment:
`ensure(eventType instanceof WalletEventType)`

Reply:
Can't be use here. 
instarceof can be used with classes(on right sides) and values with any type (leftSide) of instanceof